src/sentry/ingest/userreport.py

```diff
@@ -28,6 +39,9 @@ def save_userreport(project, report, start_time=None):
     # exist, and ideally we'd also associate that with the event
     euser = find_event_user(report, event)
 
+    if features.has("organizations:eventuser-from-snuba", project.organization):
+        find_event_user_with_snuba(event, euser)
```

Comment:
Let's wrap this in a try/except just to be safe and make sure we never break anything